src/AssetPublisher.php

```diff
@@ -244,7 +245,9 @@ private function hash(string $path): string
             return ($this->hashCallback)($path);
         }
 
-        $path = (is_file($path) ? dirname($path) : $path) . FileHelper::lastModifiedTime($path);
+        $dirname = is_file($path) ? dirname($path) : $path;
+        $iterator = new RecursiveDirectoryIterator($dirname, RecursiveDirectoryIterator::SKIP_DOTS);
+        $path = $dirname . FileHelper::lastModifiedTime($path) . iterator_count($iterator);
```

Comment:
Twice scan directory (here and in `FileHelper::lastModifiedTime`). Maybe create new method in `FileHelper`, `lastModifiedHash` or something similar?

Reply:
> Twice scan directory (here and in `FileHelper::lastModifiedTime`). Maybe create new method in `FileHelper`, `lastModifiedHash` or something similar that scan direcrtory once?

Yes - i think about that and it's a better solution. But it will affect two packages at once

Reply:
> Twice scan directory (here and in `FileHelper::lastModifiedTime`). Maybe create new method in `FileHelper`, `lastModifiedHash` or something similar that scan direcrtory once?

How about public `lastModifiedIterator` or  something like it? In case when must save bc with php 7.4 (in other way can be usefull union type) this method can be allow that iterator as arguments